src/main/java/com/stripe/param/TaxRateCreateParams.java

```diff
@@ -249,5 +282,14 @@ public Builder setPercentage(BigDecimal percentage) {
       this.percentage = percentage;
       return this;
     }
+
+    /**
+     * <a href="https://en.wikipedia.org/wiki/ISO_3166-2:US">ISO 3166-2 subdivision code</a>,
+     * without country prefix. For example, &quot;NY&quot; for New York, United States.
```

Comment:
Not a big deal, but it'd probably be better if these were rendered as non-HTML quotes for more code friendliness.